### DISCUSSION TITLE
Move developer docs to wiki

### DISCUSSION THREAD

**Discussion by aornugent** [traitecoevo/plant#432] [Ideas]
**Describe the bug**
I forget to document thinks I learn as a plant developer. Documentation always gets neglected. 

**To Reproduce**
I think @dfalster has tagged a number of documentation issues that need to be finished. 

**Expected behavior**
It's easy to add notes and we grow a deep (ie. low level) knowledge base that remains up to date. 

I think wikis might provide this. Here is a good example: https://github.com/posit-dev/positron/wiki

**Suggested solution**
We move the developer documentation for plant to the wiki.

**Comment by dfalster** [comment]
Thanks! Interesting, I remove the wiki option (we had some old stuff there) and was consolidating about the docs. I agree wiki's could be easier to update, but I really haven't seen good examples. Have you seen any good examples on GitHub? 

After a quick search, I found this page, listing some examples: https://github.com/MyHoneyBadger/awesome-github-wiki?tab=readme-ov-file

There's also the Discussions feature, which could be useful (as an alternative to issues)

https://docs.github.com/en/discussions

**Reply by dfalster** [reply]
As an experiment, I've converted @aornugent's issue to a "Discussion".  There's a feature to "Create issue from Discussion", but the best starting point is Discuss this, then relevant to issue when we agree we need to make a change. 

![Screenshot 2024-06-28 at 1 26 36 PM](https://github.com/traitecoevo/plant/assets/3589590/f65c9010-90f4-46af-8063-1ddf86d12460)

**Reply by dfalster** [reply]
I've also re-enabled the wiki feature, so that we can experiment with it

**Comment by dfalster** [comment]
@itowers1  - as someone who's worked with plant, do you have any thoughts?

**Comment by aornugent** [comment]
What I liked about the Posit example was that it was quite low level. I am only suggesting that we move this section and provide a link from the README

https://traitecoevo.github.io/plant/articles/developer_notes.html

**Comment by dfalster** [comment]
Resolved: docs now have a dedicated home at https://github.com/traitecoevo/overstorey (https://traitecoevo.github.io/overstorey/) rather than a wiki. Closing as outdated.
